src/app/types/DataResult.ts

```diff
@@ -80,6 +81,11 @@ export interface SiteDataResult extends DataResult<Site> {
   canAssignUsers: boolean;
   canUnassignUsers: boolean;
 }
+export interface SiteAreaDataResult extends DataResult<SiteArea> {
+  canCreate: boolean;
+  canAssignChargingStations: boolean;
+  canUnassignChargingStations: boolean;
+}
```

Comment:
Doesn't it also need `canAssignAssets` & `canUnassignAssets`?

Reply:
Actually that shouldn't be there, at the root level we only have `canCreate`. Will clean up `SiteDataResult` as well. 👍